crowbar_framework/app/assets/stylesheets/content/content.scss

```diff
@@ -65,6 +65,22 @@ h3.modal-title {
   font-size: 19px;
 }
 
+.installer {
+  li {
+    border-radius: 3px;
+    line-height: 2em;
+    padding-right: 5px;
+  }
+
+  .action > li > span {
+    margin-right: 5px;
+  }
+}
+
+.copyright-tag {
```

Comment:
Ah, but this was gone in the PR because the string got changed to Crowbar. Now this makes sense :-)